s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Overlay.java

```diff
@@ -0,0 +1,18 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.Data;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+@Data
+public class Overlay {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String projection;
```

Comment:
After some thought `projection` isn't needed. It can be inferred based on the shapefiles by GeoServer itself when creating the layers and datastores.
Also, I wrote the DB diagram thinking about polar stereographic, now when we mostly default to Web Mercator this doesn't have to be stored independently in the DB.